code/player/Player.cs

```diff
@@ -157,30 +212,37 @@ public override void TakeDamage(DamageInfo info)
             base.TakeDamage(info);
         }
 
-        public void RemovePlayerCorpse()
+        [ClientRpc]
+        public static void ClientOpenInspectMenu(TTTPlayer deadPlayer, bool isIdentified)
         {
-            if (PlayerCorpse != null && PlayerCorpse.IsValid())
+            InspectMenu.Instance?.InspectCorpse(deadPlayer, isIdentified);
+        }
+
+        [ClientRpc]
+        public static void ClientCloseInspectMenu()
+        {
+            if (InspectMenu.Instance?.IsShowing ?? false)
             {
-                PlayerCorpse.Delete();
-                PlayerCorpse = null;
+                InspectMenu.Instance.IsShowing = false;
             }
         }
 
         [ClientRpc]
-        public void DidDamage(Vector3 position, float amount, float inverseHealth)
+        public static void ClientDisplayIdentifiedMessage(ulong leftId, string left, ulong rightId, string right, string role)
         {
-            Sound.FromScreen("dm.ui_attacker")
-                .SetPitch(1 + inverseHealth * 1);
+            // TODO: Refactor the UI element, and provide a better interface for passing in these parameters.
+            InfoFeed.Current?.AddEntry(leftId, left, rightId, $"{right}.  Their role was {role}!", "found the body of");
         }
 
         [ClientRpc]
-        public void TookDamage(Vector3 position)
+        public void DidDamage(Vector3 position, float amount, float inverseHealth)
         {
-
+            Sound.FromScreen("dm.ui_attacker")
+                .SetPitch(1 + inverseHealth * 1);
```

Comment:
`* 1` ? :D

Reply:
No idea, just taken from FP DM gamemode, looks like just to normalize the pitch.